.travis.yml

```diff
@@ -140,7 +140,7 @@ matrix:
 
 before_install:
   - travis_retry composer require "laravel/framework:${LARAVEL_VERSION}" --no-update
-  - echo "memory_limit=2G" >> ~/.phpenv/versions/$(phpenv version-name)/etc/conf.d/travis.ini
+  - echo "memory_limit=4G" >> ~/.phpenv/versions/$(phpenv version-name)/etc/conf.d/travis.ini
```

Comment:
This is from https://github.com/bugsnag/bugsnag-laravel/pull/394 to get the tests passing on Travis